docs/Deploy-Module.md

```diff
@@ -19,80 +19,76 @@ Apart from Tachyon, you can choose to deploy computation frameworks and under fi
+7. From inside TachyonMaster node, run `ssh TachyonWorker1` to login to TachyonWorker1 without password
+8. If you don't want to play around in the cluster any more, `./stop` to destroy the cluster, 
+for virtualbox, virtual machines will be deleted, for EC2, instances will be terminated.
 
 ## Configurations
 
-There are three kinds of configuration files under `conf`. All the configuration files use [yaml syntax](http://en.wikipedia.org/wiki/YAML).
+There are two kinds of configuration files under `conf`:
 
-1. configuration of virtual machine itself in `init.yml`. 
-2. provider specific configuration, like `ec2.yml`, `openstack.yml`.
-3. software configuration, like `tachyon.yml`, `spark.yml`, `ufs.yml`(ufs means under filesystem). 
+1. provider specific configuration, like `vb.yml` for virtualbox, `ec2.yml` for AWS EC2
+2. software configuration, like `tachyon.yml`, `spark.yml`, `ufs.yml`(ufs means under filesystem). 
 
-Meanings of fields in the configurations are explained in detail in the yml files.
-
-## AWS EC2
+All the configuration files use [yaml syntax](http://en.wikipedia.org/wiki/YAML).
 
-You need to have AWS account with access to EC2. If not, sign up [here](https://aws.amazon.com/). 
-
-Set up [AWS CLI](http://docs.aws.amazon.com/AWSEC2/latest/CommandLineReference/ec2-cli-get-set-up.html) may be helpful.
+Meanings of fields in the configurations are explained in detail in the yml files.
 
-Then you need to set shell environment variables `AWS_ACCESS_KEY`
-and `AWS_SECRET_KEY`, refer to [this doc](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html). 
+If you are new to AWS EC2, we recommend you to understand comments in `ec2.yml`.
 
-Then download [key pair](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
+## FAQ
 
-Install aws vagrant plugin. To date, 0.5.0 plugin is tested.
+1. How to increase virtualbox virtual machine's memory size?
 
-    vagrant plugin install vagrant-aws
+  Increase **MachineMemory** in **conf/vb.yml**
 
-Then update configurations in `conf/ec2.yml`.
+2. How to increase Tachyon worker's memory?
 
-Update `init.yml` and software configurations. 
+  Increase **WorkerMemory** in **conf/tachyon.yml**
 
-Run `./run_aws.sh`, then `vagrant ssh TachyonMaster` or `vagrant ssh TachyonWorker#{number}`, to log into remote nodes. 
+3. How to increase HDFS capacity in AWS?
+4. How to use EBS in AWS?
+5. Why extra storage specified in Instance Type not available?
 
-You can monitor instance running state through [AWS web console](https://console.aws.amazon.com).
+  Set **Block_Device_Mapping** in **conf/ec2.yml** according to the comment there
 
-`vagrant destroy` to terminate the instances.
+6. What's PV/HVM?
 
+  see comment on **AMI** in **conf/ec2.yml**
```

Comment:
I think it might be better to just show the link explaining the two types of methods: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html